### ISSUE TITLE
Questions on GeoStory

### ISSUE THREAD

**Issue by seabilwe** [kartoza/IGRAC-GGIS#570]
Question from the support portal: https://kartoza.com/app/issue/ISS-2023-00013

Dear colleagues, we have a few questions regarding the GeoStory functionnality
1) How can you define the initial zoom level in a GeoCarousel? We changed the zoom level and location of the background map but it is not preserved after saving the GeoStory
2) How can you move a title or text holder?
3) Can you make a GeoCarousel with multiple categories of cards? Or would you need to create several GeoCarousels, one per category?

**Comment by ArnaudIGRAC** [comment]
Can anybody answer our questions?

**Comment by NyakudyaA** [comment]
@ArnaudIGRAC I will check later after my training and revert

**Comment by ArnaudIGRAC** [comment]
Come on colleagues, can you please answer my questions?

**Comment by MarinaKartoza** [comment]
Good day Arnaud. Again I apologize for our tardiness.

I am going to find the answers to your questions now and will revert back as soon as we have them.

**Comment by NyakudyaA** [comment]
@ArnaudIGRAC Can we have a quick session to walk me through these issues? I have some ideas about how to resolve some of them but I need to understand that I am interpreting this correctly

**Comment by ArnaudIGRAC** [comment]
Yes, sure! I am available all day, today or tomorrow. Please send me a Meet invite.

**Comment by ArnaudIGRAC** [comment]
Any update on these issues?
